### DISCUSSION TITLE
Renovate cannot install Go 1.21

### DISCUSSION THREAD

**Discussion by uhthomas** [renovatebot/renovate#23763] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the problem

[Go 1.21](https://tip.golang.org/doc/go1.21) has just been released, but Renovate cannot install it.

```sh
Command failed: install-tool golang 1.21.0
curl: (22) The requested URL returned error: 404 
Download failed: https://dl.google.com/go/go1.21.linux-amd64.tar.gz
```

It looks like they have changed how releases are versioned, as https://dl.google.com/go/go1.21.0.linux-amd64.tar.gz exists.

```plaintext
Starting in Go 1.21, the first release is now Go 1.N.0.
```

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by rarkins** [comment]
@Chumper @viceice needs a change in containerbase?

**Reply by viceice** [reply]
seems so, this should be an issue in containerbase base. can fix it tomorrow morning

**Reply by maxbrunet** [reply]
Opened containerbase/base#1287

**Reply by viceice** [reply, marked as answer]
fix merged, we need to wait for a release.

https://github.com/containerbase/base/actions/runs/5805545369
https://github.com/containerbase/base/releases/tag/9.9.1

**Reply by cep21** [reply]
WIth github issues, I would know the problem is fixed for renovate when the issue is closed.  Since this is a discussion, how will I know: will someone post an update here?

**Reply by viceice** [reply]
self hosted users need to update to latest renovate `36.40.3` or later

**Reply by viceice** [reply]
the hosted app is updated too, I've checked a log of one of our repos

**Reply by renandotcorrea** [reply]
I have the same problem with the latest version of Renovate on docker hub. I'm using the Gitlab self-hosted mode.

**Reply by viceice** [reply]
please open a new discussion with more details
